IHP/IDE/Data/Controller.hs

```diff
@@ -27,8 +28,9 @@ instance Controller DataController where
     action ShowTableRowsAction { tableName } = do
         connection <- connectToAppDb
         tableNames <- fetchTableNames connection
+        primaryKeyFields <- tablePrimaryKeyFields connection tableName
 
-        rows :: [[DynamicField]] <- PG.query connection "SELECT * FROM ? ORDER BY id" (PG.Only (PG.Identifier tableName))
+        rows :: [[DynamicField]] <- fetchRows connection tableName
```

Comment:
By using `fetchRows` here, we lose the `ORDER BY id`. This has been added here to avoid rows jumping around when using the data editor when updating a value of a row.

When we don't order by the primary key, we will get the results in the order they're read by postgres from the disk. This order usually changes when doing running an `UPDATE` statement. Maybe we can query the primary keys before doing this query and then add them in a `ORDER BY` clause?

Reply:
Forgot what I've written above. Just saw that `fetchRows` is already doing that 👍  very good job